official/vision/beta/projects/panoptic_maskrcnn/configs/panoptic_maskrcnn.py

```diff
@@ -60,6 +75,8 @@ class PanopticMaskRCNN(maskrcnn.MaskRCNN):
   include_mask = True
   shared_backbone: bool = True
   shared_decoder: bool = True
+  panoptic_segmentation_generator: PanopticSegmentationGenerator = \
```

Comment:
Can you add an option disable having this branch?

Reply:
A flag that can decide whether or not to have this as part of the outputs during eval.

Reply:
remove \